sql/5_0_0-to-5_0_1_upgrade.sql

```diff
@@ -661,3 +661,7 @@ DELETE FROM list_options WHERE list_id = 'lists' AND option_id = 'lbfnames';
 DELETE FROM list_options WHERE list_id = 'transactions';
 DELETE FROM list_options WHERE list_id = 'lists' AND option_id = 'transactions';
 #EndIf
+
+#IfRow2D layout_options source F edit_options %0%
+UPDATE layout_options SET edit_options = REPLACE(edit_options, '0', '3') WHERE edit_options LIKE '%0%';
```

Comment:
Why doesn't '0' work? Would really prefer to solve the underlying problem, especially as '0' is a good mnemonic.

Reply:
I didn't write a new code.. @sjpadgett do you Know why?
But I know that zero equal to false in some situations (http://php.net/manual/en/types.comparisons.php) so why not simple change it?

Reply:
it  fails because I used empty() in isOption var conditional instead of isset(). empty() is false for null and '0' and 0 so Read Only is never eval and always returns false as I indicated in previous post.
I'll post a PR after I test a fix.

Reply:
OK so don't change 0 to 3 here. Should be an easy fix.

Reply:
Yep, my boo boo! Didn't handle the json as well as I should have but, coming for the world of c, null is our friend. :) and '0' should stay
Easier I post PR which is coming soon...